lib/index.js

```diff
@@ -19,6 +72,10 @@ function onlyFiles(schemas) {
 function getAllObjects(baseQuery)  {
   var allObjects = [];
   var next = function(startIndex) {
+    if (startIndex > 10000) {
```

Comment:
I should use a variable for this: (`var maxParseSkip = 10000`)

The purpose of this is to work around Parse's limitation of skipping more than 10k. If you call `query.skip(10001)` you'll get a response of `{"code":154,"error":"Skips larger than 10000 are not allowed"}`.